validphys2/src/validphys/scripts/vp_comparefits.py

```diff
@@ -240,7 +243,29 @@ def get_config(self):
             c = yaml_safe.load(f)
         c.update(self.complete_mapping())
         return self.config_class(c, environment=self.environment)
+    
+    def check_identical_theory_cuts_covmat(self):
+        args = self.args
+        l = self.environment.loader
+        resultspath = l.resultspath
+        
+        current_runcard_path = str(resultspath) + "/" + str(args['current_fit']) + "/filter.yml" 
+        reference_runcard_path = str(resultspath) + "/" + str(args['reference_fit']) + "/filter.yml"
```

Comment:
Since at this point you already have the loader, you can use `l.check_fit(fitname)` to get a reference to the fit (and with `fitname.as_input()`... I think, you get already most of the information you need).

The relevance of using `.check_fit` instead of looking directly at the path is that at this point the fit might not have been downloaded yet. `.check_fit` will ensure that they are downloaded and available (and at that point you will get a reference to them, so no need to generate yourself the reference or the load).


for future reference, when you are getting a path you can do directly:
```suggestion
        reference_runcard_path = resultspath / str(args['reference_fit']) / "filter.yml"
```
and so is python who will build the path instead of you. It's not that important for us but this line wouldn't work in systems for which the path is `\` .